.gitlab/ci/.gitlab-ci.bucket-upload.yml

```diff
@@ -708,6 +708,11 @@ sync-buckets-between-projects:
         echo "Skipping syncing buckets in test upload-flow"
         exit 0
       fi
+      
+      if [[ $TEST_XDR_ENV == "true" ]]; then
```

Comment:
sure it's `_ENV` and not `_PREFIX`? (just asking for sanity, as I didn't see `_ENV` elsewhere)

Reply:
yes :) 2 different variables